src/containers/shared/components/Transaction/NFTokenAcceptOffer/test/NFTokenAcceptOfferSimple.test.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { BrowserRouter as Router } from 'react-router-dom'
+import { mount } from 'enzyme'
+import { I18nextProvider } from 'react-i18next'
+import { Simple as NFTokenAcceptOffer } from '../Simple'
+import transactionBuy from './mock_data/NFTokenAcceptOffer_Buy.json'
+import transactionSell from './mock_data/NFTokenAcceptOffer_Sell.json'
+import transactionFailure from './mock_data/NFTokenAcceptOffer_Failure.json'
+import summarizeTransaction from '../../../../../../rippled/lib/txSummary'
+import i18n from '../../../../../../i18nTestConfig'
+
+describe('NFTokenAcceptOffer', () => {
+  it('handles NFTokenAcceptOffer Buy simple view ', () => {
+    const wrapper = mount(
+      <I18nextProvider i18n={i18n}>
+        <Router>
+          <NFTokenAcceptOffer
+            data={summarizeTransaction(transactionBuy, true).details}
+          />
+        </Router>
+      </I18nextProvider>,
+    )
+    expect(wrapper.find('[data-test="token-id"] .value')).toHaveText(
+      '000800006203F49C21D5D6E022CB16DE3538F248662FC73C29ABA6A90000000D',
+    )
+    expect(wrapper.find('[data-test="offer-id"] .value')).toHaveText(
+      '8278760A246D4464EE701D503091B9DB0D9790DD2BBE9CAABCA45B04A1A25B6B',
+    )
+    expect(wrapper.find('[data-test="amount"] .value')).toHaveText(
+      '0.0001 XRP',
```

Comment:
You should add the utf-8 character code instead.  Many of the other tests do that already.